Project.toml

```diff
@@ -4,17 +4,19 @@ authors = ["Seth Axen <seth.axen@gmail.com>", "Ronny Bergmann <manopt@ronnybergm
 version = "0.1.0"
 
 [deps]
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Einsum = "b7d42ee7-0b51-5a75-98ca-779d3107e4c0"
```

Comment:
Isn't `Tensors.jl` only good for some small tensors? https://kristofferc.github.io/Tensors.jl/stable/man/constructing_tensors/#Constructing-tensors-1

Reply:
I would *love* to use `Tensors.jl`, but yes, it only supports up to dimension 3 right now, and I don't see any issues for supporting higher-dimensional tensors. If you know of any other good fits here, I'd be interested. Most of these tensors/tensor-like objects have symmetries/sparsities that it would be nice to explicitly handle.